shell/platform/tizen/BUILD.gn

```diff
@@ -111,6 +113,7 @@ source_set("flutter_tizen") {
     "evas",
     "flutter_engine",
     "GLESv2",
+    "feedback",
```

Comment:
I don't know if `libfeedback.so` exists in the TV rootstrap, but if not, you have to separate this dependency (and the implementation) like https://github.com/flutter-tizen/engine/pull/65#issuecomment-827318857.

Reply:
I have changed BUILD.gn, so that `feedback` is only included in builds for wearable and mobile profiles.